dotcom-rendering/src/web/server/frontToHtml.tsx

```diff
@@ -17,12 +24,47 @@ interface Props {
 	front: DCRFrontType;
 }
 
+const decideFormat = (NAV: NavType) => {
+	// We decide the `theme` of the format based on the `currentNavLink` as this is used in
+	// <Nav /> to decide which pillar should be selected
+	const { currentNavLink } = NAV;
+
+	// Is the `currentNavLink` a pillar?
+	const themeFromCurrentLink = findPillar(currentNavLink);
+
+	// Is the `currentNavLink` in one of the children of the pillar?
+	const themeFromSubNav = NAV.pillars.find((pillar) => {
+		// Annoyingly "Football" appears in "News" and "Sport" pillars, so we exclude this case in "News"
+		// As "Football" is always "Sport". You can see the corresponding `frontend` code here:
+		// https://github.com/guardian/frontend/blob/main/common/app/navigation/Navigation.scala#L141-L143
+		if (
+			pillar.pillar === ArticlePillar.News &&
+			currentNavLink === 'Football'
+		) {
+			return false;
+		}
+
+		return pillar.children?.some((child) => {
+			return child.title === currentNavLink;
+		});
+	})?.pillar;
+
+	const theme = themeFromCurrentLink ?? themeFromSubNav ?? ArticlePillar.News;
+
+	return {
+		display: ArticleDisplay.Standard,
+		design: ArticleDesign.Standard,
+		theme,
+	};
```

Comment:
Instead of trying to force fronts into the 'ArticleFormat' type, would it be more approriate to have the NAV component take some kind of 'theme' type? I'm not sure if there is a good future refactor, but I'm not sure that our nav system should be dependant on article format, when we will (and do) have other content types which aren't articles.

Reply:
Very good point. Also, I know next to nothing about format, but worth noting that `tone/comment` articles get their `opinion` pillar, so nav highlight, from `Comment is free (Blog)` tag (coz `tone/comment` alone can also be under `Sport`, `Lifestyle` etc).

Reply:
I completely agree. The nav is something accessible to the whole of our estate. We could start leaning into that on this PR if you fancy? 

I imagine we could have something like:
```
getArticleGlobalPillar(theme: ArticleTheme): Pillar
getFrontGlobalPillar(front: DCRFront): Pillar
```

Reply:
This was captured in #7562 after a discussion in https://github.com/guardian/dotcom-rendering/pull/7507#discussion_r1162642525

Reply:
@jamesgorrie you may find this function (and others in that file) useful? https://github.com/guardian/dotcom-rendering/blob/801030d40ce4dbfe2e3a9283c06effa04b7010e5/apps-rendering/src/articleFormat.ts#L90-L115